src/robot_follower_node.cpp

```diff
@@ -0,0 +1,200 @@
+#include <mw_maxl_planner/mw_maxl_planner_ros1.h>
```

Comment:
put meta info, 
for example:
file name:
brief: ( description of the purpose of the file)
author: ( name and email)
date: 
company: